src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2014.ProvideDynamicMessageContent_Invalid.sarif

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "version": "1",
+          "rules": [
+            {
+              "id": "TST0001",
+              "fullDescription": {
+                "text": "This is a test."
+              },
+              "messageStrings": {
+                "NoPlaceholders": {
+                  "text": "This message does not contain dynamic content."
+                }
+              },
+              "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
```

Comment:
>helpUri [](start = 15, length = 7)

I'm on a kick to make the test files as small as possible by removing all optional properties. Let's get rid of `helpUri`.